tzrec/main.py

```diff
@@ -746,6 +750,9 @@ def _script_model(
             logger.info(f"Model Outputs: {result_info}")
 
             export_model_trt(model, data_cuda, save_dir)
+        elif is_aot():
+            data_cuda = batch.to_dict(sparse_dtype=torch.int64)
```

Comment:
the data type is same in cpu and gpu, it can use the same data_cuda in export/trt_export/aot_export